puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pipelines/foreman-pipeline.groovy

```diff
@@ -38,12 +42,13 @@ pipeline {
         }
         stage('Install Test') {
             agent { label 'el' }
+
             steps {
                 script {
                     def pipelines = foreman_server_distros.collect { os ->
                         [
                             name: os,
-                            job: "foreman-${foreman_version}-${os}-release-test",
+                            job: "foreman-${foreman_version}-${os}-install-test",
```

Comment:
Will you rename all jobs in CentOS CI as well? I don't see that in this PR.

Reply:
Ahh yea, that was a leftover from breaking changes out. Will put tha tback for this PR.